Makefile

```diff
@@ -25,12 +25,27 @@ install: _output/bin/$(LOCAL_OS)/bootkube
 
 _output/bin/%: $(GOFILES)
 	mkdir -p $(dir $@)
-	GOOS=$(word 1, $(subst /, ,$*)) go build -ldflags "$(LDFLAGS)" -o $@ github.com/kubernetes-incubator/bootkube/cmd/$(notdir $@)
+	GOOS=$(word 1, $(subst /, ,$*)) go build $(GOFLAGS) -ldflags "$(LDFLAGS)" -o $@ github.com/kubernetes-incubator/bootkube/cmd/$(notdir $@)
 
 _output/release/bootkube.tar.gz: _output/bin/linux/bootkube _output/bin/darwin/bootkube _output/bin/linux/checkpoint
 	mkdir -p $(dir $@)
 	tar czf $@ -C _output bin/linux/bootkube bin/darwin/bootkube bin/linux/checkpoint
 
+run-%: GOFLAGS = -i
+run-%: clean-vm-% _output/bin/linux/bootkube _output/bin/$(LOCAL_OS)/bootkube
+	@until (cd hack/$*-node && vagrant up 2>/dev/null);\
+	do \
+		echo "Vagrant not ready."; \
+		sleep 5; \
+	done;
+	@cd hack/$*-node && ./bootkube-up
+	@echo "Bootkube ready"
+
+clean-vm-single:
```

Comment:
It's hacky but the named targets were taking precedence over the wildcard ones. Any idea of a better way to do this?